sample/client/browser-sample.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { authManager } from "./authConstants";
+
+const subscriptionId = "00977cdb-163f-435f-9c32-39ec8ae61f4d";
+authManager.getToken().then((t) => {
```

Comment:
if `getToken()` throws, as we know it can throw due to multiple reasons, do we capture the error and output it to the page? basically, do we need to have this chain -- `getToken().then().catch()` ?